app/src/androidTest/java/org/simple/clinic/encounter/EncounterRepositoryAndroidTest.kt

```diff
@@ -314,4 +314,35 @@ class EncounterRepositoryAndroidTest {
       assertThat(encounter.deletedAt).isNull()
     }
   }
+
+  @Test
+  fun deleting_an_encounter_should_check_for_any_associated_active_observations(){
```

Comment:
Curious, what do we do after checking for active observations?

Reply:
We basically want to verify that when deleting an encounter we should check that it only gets if there are no observations associated.
I will reword this test case.